.github/workflows/docs.yml

```diff
@@ -3,7 +3,19 @@ name: Documentation
 on:
    push:
       branches: [ RC_1_2 RC_2_0 master ]
+      paths:
+      - docs/**
+      - include/**
+      - examples/**
+      - tools/**
+      - bindings/python/simple_client.py
    pull_request:
+      paths:
+      - docs/**
+      - include/**
+      - examples/**
+      - tools/**
+      - bindings/python/simple_client.py
```

Comment:
I don't think `simple_client.py` affects the documentation. Neither does `examples` or `tools`

Reply:
The docs include other files, including `simple_client.py` and various stuff from `examples`. I thought something included `tools` but I can't find it now, removed.